packages/carbon-graphs/src/js/controls/Gantt/helpers/eventHelpers.js

```diff
@@ -135,9 +135,10 @@ const loadEventInput = (inputJSON) => {
  * @param {object} scale - d3 scale for Graph
  * @param {object} gantt - Graph config object for the content.
  * @param {object} trackGroupPath - Container for the track
+ * @param {object} legendSVG - d3 element path of the legend from the parent control
  * @returns {undefined} - returns nothing
  */
-const reflowEvents = (config, scale, gantt, trackGroupPath) => {
+const reflowEvents = (config, scale, gantt, trackGroupPath, legendSVG) => {
```

Comment:
`legendSVG` seems to be taken in as an argument but is not used in the function.  Should this be passed to the `drawDataPoints` function call on line 149 where it was added as a parameter?

Reply:
Removed it here https://github.com/cerner/terra-graphs/pull/18/commits/5b93cdca28a15258ed440acc42cd20fb6f8f3c62.